cmd/relay/auth.go

```diff
@@ -21,10 +30,37 @@ func NewAuthCommand() *cobra.Command {
 
 func NewLoginCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "login",
+		Use:   "login [email]",
```

Comment:
It will be important (until we have proper service accounts) to have a `--password-stdin` flag that expects the literal password on standard input instead of prompting. Not sure if that's something you have planned or not. The old code had it implemented so it should be pretty easy to copy from there if needed.